### PR TITLE
Add conf2kube Formula

### PR DESCRIPTION
**Pull request by antoineco**
Reopening PR #46841 now that the repo passes the audit

### CODE REVIEW

File: Library/Formula/conf2kube.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Conf2kube < Formula
+  desc "Create Kubernetes secrets from configuration files"
+  homepage "https://github.com/kelseyhightower/conf2kube"
+
```
Comment (bfontaine) [inline Library/Formula/conf2kube.rb]: Please remove this empty line.

File: Library/Formula/conf2kube.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "4462ac150059653c83235cb9c57ebf019aa377ad673d46cc6dbbc73a726cb732"
+  head "https://github.com/kelseyhightower/conf2kube.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", "-o", "conf2kube"
+    bin.install "conf2kube"
```
Comment (bfontaine) [inline Library/Formula/conf2kube.rb]: You can merge these two lines with `system "go", "build", "-o", "#{bin}/conf2kube"`.

File: Library/Formula/conf2kube.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  test do
+    shell_output("#{bin}/conf2kube -h", 2)
```
Comment (bfontaine) [inline Library/Formula/conf2kube.rb]: Can’t we write a test that does something more substantial than e.g. `-h` or `-v`?

**Comment by DomT4** [comment]
Ping on Baptiste's comments?

**Comment by antoineco** [comment]
Sorry, totally missed that one! I'll push the suggested changes

File: Library/Formula/conf2kube.rb (hunk 3 above)
Comment (antoineco) [inline Library/Formula/conf2kube.rb]: Not sure what would be the preferred way for a simple CLI. I could run the command on a file and check the output but would I have to ship this file with the Formula?

**Comment by antoineco** [comment]
The condition changed to >= 50 stars, closing

**Comment by DomT4** [comment]
I wouldn't have actually been unhappy to have considered this for merge, personally. You submitted it when the conditions were lower and you're a whole 6 stars off even the new eligibility requirements.

**Comment by antoineco** [comment]
Should be fine for now, I'll keep an eye on the project, see if it gets more traction and a new release, then I'll resubmit it.

**Pull request closed without merging**